docs/tutorial.rst

```diff
@@ -198,8 +186,10 @@ During training we will want to monitor the performance of our model on
 a separate set of examples. Let's create a new data stream for that.
 
 >>> mnist_test = MNIST("test")
->>> data_stream_test = DataStream(mnist_test, iteration_scheme=SequentialScheme(
-...     mnist_test.num_examples, batch_size=1024))
+>>> data_stream_test = DataStream.default_stream(
```

Comment:
This good catch should have ideally been done by Travis. Can we somehow empty Travis cache? Now it is really dangerous: changes in Fuel are ignored when testing Blocks master and PRs.